### PR TITLE
feat: add GenDigital implementation

### PR DESCRIPTION
**Pull request by skynet2**

### CODE REVIEW

**Comment by sudeshrshetty** [comment]
@BigBlueHat please review and merge

**Comment by BigBlueHat** [comment]
@skynet2 @sudeshrshetty same thing needed as on https://github.com/w3c/vc-test-suite-implementations/pull/84#issuecomment-2515626476

The `id` needs to be a URI--ideally a `did:key`.

**Comment by skynet2** [comment]
hi @BigBlueHat ,
Updated, please review.

Thanks,
Stas

**Review by PatStLouis — APPROVED** [review]
@skynet2 fyi the BitstringStatusList test suites have migrated to the w3c and don't need to be duplicated here as you have included them in the w3c implementations.

**Comment by BigBlueHat** [comment]
> @skynet2 fyi the BitstringStatusList test suites have migrated to the w3c and don't need to be duplicated here as you have included them in the w3c implementations. Otherwise this looks good!

Can you remove the `BitstringStatusList` entries? Those are covered by your other PR: https://github.com/w3c/vc-test-suite-implementations/pull/84

Otherwise, this looks good!

File: implementations/GenDigital.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "id": "did:web:fd5e2e37-8ee6-424e-92df-28d34fe4e0a5.did-dev3.dev.dts-dsa.com",
+      "endpoint": "https://krakend-k8s-dev3.dev.dts-dsa.com/vcs/issuer/profiles/78980eb4-3862-4a78-8b65-b158b580aeec/v1.latest/credentials/issue",
+      "tags": ["vc-api"]
+    },
+    {
+      "id": "did:web:fd5e2e37-8ee6-424e-92df-28d34fe4e0a5.did-dev3.dev.dts-dsa.com",
+      "endpoint": "https://krakend-k8s-dev3.dev.dts-dsa.com/vcs/issuer/profiles/21dc85e3-a9e9-4304-bc77-88874fa3f37d/v1.latest/credentials/issue",
+      "tags": ["BitstringStatusList", "Revocation"]
+    }
```
Comment (BigBlueHat) [inline implementations/GenDigital.json:14]:
```suggestion
    }
```

File: implementations/GenDigital.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "id": "did:web:481ca770-fca8-4425-97be-96f057d6952a.did-dev3.dev.dts-dsa.com",
+      "endpoint": "https://krakend-k8s-dev3.dev.dts-dsa.com/vcs/verifier/profiles/25c3a9cd-2a45-418e-9200-f7e8a24d1a43/v1.latest/credentials/verify",
+      "tags": ["vc-api"]
+    },
+    {
+      "id": "did:web:481ca770-fca8-4425-97be-96f057d6952a.did-dev3.dev.dts-dsa.com",
+      "endpoint": "https://krakend-k8s-dev3.dev.dts-dsa.com/vcs/issuer/profiles/f3c5e456-2528-4821-9516-e062392f6ead/v1.latest/credentials/issue",
+      "tags": ["BitstringStatusList", "Revocation"]
+    }
```
Comment (BigBlueHat) [inline implementations/GenDigital.json:26]:
```suggestion
    }
```

**Review by BigBlueHat — CHANGES_REQUESTED** [review]

**Comment by skynet2** [comment]
hi @BigBlueHat ,
Thanks for the patch. I applied it.

Thanks,
Stas

**Comment by fqutishat** [comment]
Hi @BigBlueHat  can you please review the PR we made the change that requested

**Review by PatStLouis — APPROVED** [review]

**Comment by sudeshrshetty** [comment]
@PatStLouis Can you please generate latest interop report for vc-api issuer & verifier test suites.

**Pull request merged**